packages/appstore/src/components/app.tsx

```diff
@@ -15,6 +16,25 @@ type TAppProps = {
     };
 };
 
+const DATADOG_APP_ID = process.env.DATADOG_APPLICATION_ID ? process.env.DATADOG_APPLICATION_ID : '';
+const DATADOG_CLIENT_TOKEN = process.env.DATADOG_CLIENT_TOKEN ? process.env.DATADOG_CLIENT_TOKEN : '';
+
+datadogRum.init({
+    applicationId: DATADOG_APP_ID,
+    clientToken: DATADOG_CLIENT_TOKEN,
+    site: 'datadoghq.com',
+    service: 'deriv.com-static-site',
+    env: 'qa',
+    // Specify a version number to identify the deployed version of your application in Datadog
+    // version: '1.0.0',
+    sessionSampleRate: 100,
```

Comment:
Before merging, I think `sessionSampleRate: 100` should be changed to `sessionSampleRate: 1`?

Reply:
Changed :+1: